package.json

```diff
@@ -1,23 +1,25 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "scripts": {
-    "build": "turbo run build",
-    "dev": "turbo run dev --parallel",
     "lint": "turbo run lint",
     "test": "turbo run test",
-    "clean": "turbo run clean && rm -rf node_modules",
-    "format": "prettier --write \"**/*.{js,md}\""
+    "prettier": "prettier -w .",
+    "prepare": "husky install",
+    "eslint": "cross-env TIMING=1 eslint --max-warnings 0 --config .eslintrc.js"
   },
   "devDependencies": {
-    "prettier": "^2.5.1",
-    "turbo": "^1.1.3"
+    "@babel/core": "^7.17.8",
```

Comment:
Where is `@babel/core` used? Where is `typescript` used?

Reply:
They are peer deps of the style guide

Reply:
```
devDependencies:
- @babel/core 7.17.8
- @vercel/style-guide 3.0.0
+ @vercel/style-guide 3.0.0
- typescript 4.6.3

 WARN  Issues with peer dependencies found
.
└─┬ @vercel/style-guide
  ├─┬ @babel/eslint-parser
  │ └── ✕ missing peer @babel/core@>=7.11.0
  ├─┬ @typescript-eslint/eslint-plugin
  │ ├── ✕ missing peer typescript@"*"
  │ └─┬ @typescript-eslint/type-utils
  │   ├── ✕ missing peer typescript@"*"
  │   └─┬ @typescript-eslint/utils
  │     └─┬ @typescript-eslint/typescript-estree
  │       ├── ✕ missing peer typescript@"*"
  │       └─┬ tsutils
  │         └── ✕ missing peer typescript@">=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta"
  └─┬ @typescript-eslint/parser
    └── ✕ missing peer typescript@"*"
Peer dependencies that should be installed:
  @babel/core@>=7.11.0  typescript@>=2.8.0  
```

Reply:
We're talking about this now. So, as I understand, `pnpm` isn't happy that some of `@vercel/style-guide`'s dependencies don't have their peer dependencies installed.

It probably makes sense for us to add `@babel/core` as a dependency of the style guide, but TypeScript feels more dangerous. @CodyBrouwers suggested we could possibly seperate that part of the style guide into a different package. We'll give it some more thought.

Reply:
Peer dependencies are the bane of my existence 😅 happy to keep them as dev deps for now though. Wdyt @styfle ?

Reply:
What happens if they're not installed explicitly? The style-guide won't work?

Reply:
I'm going to move forward with including the dependencies. When @vercel/style-guide is updated i can remove them.